tests/ha/start_up.sh

```diff
@@ -13,10 +26,13 @@ while true; do
     echo -n .
   fi
 done
-docker ps
+echo ✅ ceph is healthy
+echo ℹ️  Running processes of services
+docker-compose top
 
-# Send nvme-gw create for both gateways
-for i in $(seq 2); do
+echo ℹ️  Send nvme-gw create for all gateways
+for i in $(seq $SCALE); do
   GW_NAME=$(docker ps --format '{{.ID}}\t{{.Names}}' | grep -v discovery | awk '$2 ~ /nvmeof/ && $2 ~ /'$i'/ {print $1}')
+  echo  📫 nvme-gw create gateway: \'$GW_NAME\' pool: \'rbd\', group: \'\' \(empty string\)
```

Comment:
what if we use a different pool than "rbd"?

Reply:
I guess in such a case we'll need manual intervention anyway

Reply:
Thank you for review 👍. The fix is for the "demo" environment, no effect on pytests environment. However good point on pool being hardcoded, added ability to override the gateway pool using RBD_POOL variable, see [here](https://github.com/ceph/ceph-nvmeof/compare/775106f90c599de82aceac59a4460441ff1abb42..ea28914694fe014945dec984f2401bfef5d4497c).